src/calculator/SFA.DAS.ProviderPayments.Calc.PaymentsDue.IntegrationTests/FinishedOnTime/WhenMakingDuePaymentsForNonDasLearners.cs

```diff
@@ -14,6 +15,23 @@ public void Arrange()
             TestDataHelper.Clean();
         }
 
+        private static readonly EarningsToPaymentEntity[] _earningsToPayments = new[]
```

Comment:
I could be wrong but this language feature probably isn't supported on the build server?